2-ui/2-events/03-event-delegation/article.md

```diff
@@ -1,19 +1,19 @@
 
-# Event delegation
+# Delegación de eventos
 
-Capturing and bubbling allow us to implement one of most powerful event handling patterns called *event delegation*.
+La captura y el burbujeo nos permiten implementar uno de los más poderosos patrones de manejo de eventos llamado *delegación de eventos*.
 
-The idea is that if we have a lot of elements handled in a similar way, then instead of assigning a handler to each of them -- we put a single handler on their common ancestor.
+La idea es que si tenemos montones de elementos manejados de manera similar podemos, en lugar de asignar un manejador a cada uno de ellos, poner un único manejador a su ancestro común.
```

Comment:
> En los otros artículos está como "controlador". Vamos a confundir al lector si cambiamos las palabras artículo en artículo.
> […](#)
> On Mon, Aug 31, 2020, 21:35 Valentina VP ***@***.***> wrote: ***@***.**** commented on this pull request. ------------------------------ In 2-ui/2-events/03-event-delegation/2-sliding-tree/task.md <[#353 (comment)](https://github.com/javascript-tutorial/es.javascript.info/pull/353#discussion_r480501118)> : > -- Only one event handler (use delegation) -- A click outside the node title (on an empty space) should not do anything. +- Solamente un manejador de eventos (usa delegación) Lo mismo que mi comentario anterior. Parece que a elección de @joaquinelio <https://github.com/joaquinelio> prefiere "manejador" — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <[#353 (comment)](https://github.com/javascript-tutorial/es.javascript.info/pull/353#discussion_r480501118)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AMLHWQS7GFZMWMREHA23KWDSDQ6U7ANCNFSM4QG36QZA> .

Para ello quería unificar el glosario, pero como dijiste, hay palabras que quedan a elección del autor. NO PODEMOS FORZAR a los colaboradores. Quizás **"handler"** es una palabra que debe estar en la lista **"elección del autor"** en el glosario.